src/frontend/src/instance/distributed.rs

```diff
@@ -626,28 +627,16 @@ impl DistInstance {
 
     async fn handle_dist_delete(
         &self,
-        request: DeleteRequest,
+        request: DeleteRequests,
         ctx: QueryContextRef,
     ) -> Result<Output> {
-        let catalog = ctx.current_catalog();
-        let schema = ctx.current_schema();
-        let table_name = &request.table_name;
-        let table_ref = TableReference::full(catalog, schema, table_name);
-
-        let table = self
-            .catalog_manager
-            .table(catalog, schema, table_name)
-            .await
-            .context(CatalogSnafu)?
-            .with_context(|| TableNotFoundSnafu {
-                table_name: table_ref.to_string(),
-            })?;
-
-        let request = common_grpc_expr::delete::to_table_delete_request(request)
-            .context(ToTableDeleteRequestSnafu)?;
-
-        let affected_rows = table.delete(request).await.context(TableSnafu)?;
-        Ok(Output::AffectedRows(affected_rows))
+        let deleter = DistDeleter::new(
+            ctx.current_catalog().to_string(),
+            ctx.current_schema().to_string(),
+            self.catalog_manager(),
+        );
+        let affected_rows = deleter.grpc_delete(request).await?;
```

Comment:
Make `grpc_delete` return the `usize` type directly to avoid casting.

Reply:
Done: [222c098](https://github.com/GreptimeTeam/greptimedb/pull/2150/commits/222c0985a930c41a00050f1a7b05938bef7ab9cd)

Please let me know if there are any errors or omissions.